vendors/xilinx/boards/microzed/aws_demos/config_files/FreeRTOSConfig.h

```diff
@@ -165,7 +165,10 @@ extern void vLoggingPrint( const char * pcMessage );
 
 /* Map the logging task's printf to the board specific output function. */
 #include "xil_printf.h"
-#define configPRINT_STRING( X )    xil_printf( X );
+#define configPRINT_STRING( X ) \
+	xil_printf( X ); \
+	vTaskDelay( 20 )
+
```

Comment:
This is a very important change for debugging: `xil_printf()` gets overflown with logging messages and 75% of the messages got lost. A short pause of 20 clock ticks makes sure all messages are sent.